style-guides/scss.md

```diff
@@ -0,0 +1,123 @@
+<script>{
+	"title": "Chassis SCSS Style Guide"
+}</script>
+
+This page outlines the style guide for SCSS for the jQuery Chassis project.
+
+## Comments
+
+SCSS comments should follow the valid CSS comment guidelines outlined in the jQuery contribute CSS style guide.
```

Comment:
As I recall, the meeting on 05-05-2015, we decided everything was going to be valid css comments, thus issue #78.  I don't recall changing directions since then.